changelog/1.17.3.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "1.17.3"
+description: "Released on 04/08/21"
+---
+
+### Breaking Changes ❗
+
+There are no breaking changes in 1.17.3.
+
+### Features ✨
+
+There are no new features in 1.17.3.
+
+### Bug Fixes 🐛
+
+- infra: Fixes an issue whereby workspace application icons appear broken and
+  the workspace provider ping indicator always displays as an error. The
```

Comment:
> Fixed issue where icons in the **Applications** section of the **Environment Overview** page appear broken and the workspace provider **ping indicator** always...

Also, are we officially using workspace now (e.g., workspace application icons)?

Reply:
Ah - not until 1.18...good catch :p